CHANGELOG.md

```diff
@@ -54,6 +54,9 @@
 
 ### Bug Fixes
 
+* Plist: Fix issue with Array in the Inline templates.
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/SwiftGen/SwiftGen/blob/850bb4c91f7289e48da6c0ca028773de29885af0/CHANGELOG.md#L57"></span>Line describing your entry should end with a period and 2 spaces.</td>
    </tr>
  </tbody>
</table>